Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM python:3-slim
 
+ENV IN_CONTAINER=true
+ENV OS_OUTSIDE_CONTAINER="Linux/Red Hat Enterprise Linux 8.8 (Ootpa)"
```

Comment:
This should be an argument, so user can setup the image with specific OS and SHELL names.

Reply:
In the upcoming commit I'll leave it as ENV variable that is in the config.py DEFAULT_CONFIG.
Reason being that it might be handy to leave it interchangeable, e.g.:

- All of a sudden you need to work on a server with an os / shell you're very unfamiliar with. You can temporarily overwrite your os and shell name of your often used shellgpt to mock this environment, without having to install on that server / install in a similar virtual local environment
- Similar to above, if I'd like to use shellgpt containers for multiple setups: RHEL, Debian, Ubuntu, powershell, MacOs, ... I can run the same image each time, as long as I overwrite the OS and shell to mock these envs. If I had to make a different image each time that'd be a waste of diskspace and image build time in my opinion.